tools/c7n_azure/c7n_azure/resources/sqlserver.py

```diff
@@ -136,6 +137,77 @@ def __call__(self, i):
         return super(AzureADAdministratorsFilter, self).__call__(i['properties']['administrators'])
 
 
+@SqlServer.filter_registry.register('vulnerability-assessment')
+class VulnerabilityAssessmentFilter(Filter):
+    """
+    Filter sql servers by whether they have recurring vulnerability scans
+    enabled.
+
+    :example:
+
+    Find SQL servers without vulnerability assessments enabled.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: sql-server-no-va
+            resource: azure.sql-server
+            filters:
+              - type: vulnerability-assessment
+                enabled: false
+
+    """
+
+    schema = type_schema(
+        'vulnerability-assessment',
+        required=['type', 'enabled'],
+        **{
+            'enabled': {"type": "boolean"},
+        }
+    )
+
+    log = logging.getLogger('custodian.azure.sqldatabase.VulnerabilityAssessmentFilter')
+
+    def __init__(self, data, manager=None):
+        super(VulnerabilityAssessmentFilter, self).__init__(data, manager)
+        self.enabled = self.data.get('enabled')
+
+    def process(self, resources, event=None):
+        resources, exceptions = ThreadHelper.execute_in_parallel(
+            resources=resources,
+            event=event,
+            execution_method=self._process_resource_set,
+            executor_factory=self.executor_factory,
+            log=log
+        )
+        if exceptions:
+            raise exceptions[0]
+        return resources
+
+    def _process_resource_set(self, resources, event=None):
+        client = self.manager.get_client()
+        result = []
+        for resource in resources:
+            if 'c7n:vulnerability_assessment' not in resource['properties']:
+                va = list(client.server_vulnerability_assessments.list_by_server(
+                    resource['resourceGroup'],
+                    resource['name']))
+
+                # there can only be a single instance named "Default".
+                if va:
+                    resource['c7n:vulnerability_assessment'] = \
+                        va[0].serialize(True).get('properties', {})
+                else:
+                    resource['c7n:vulnerability_assessment'] = {}
+
+            if resource['c7n:vulnerability_assessment']\
+                    .get('recurringScans', {})\
+                    .get('isEnabled') == self.enabled:
```

Comment:
`.get('isEnabled', False)` ?

Reply:
I was thinking if it comes back null you'd want it to trigger either way, but your right null can just be assumed the same as false I guess.